src/main/resources/data/invoice-lines/123invoicenumber45-1.json

```diff
@@ -0,0 +1,39 @@
+{
+  "id": "e0d08448-343b-118a-8c2f-4fb50248d672",
+  "adjustments": [
+    {
+      "description": "Service Fee",
+      "type": "Amount",
+      "value": 5.00,
+      "prorate":"Not prorated",
+      "relationToTotal":"In addition to"
+    },
+    {
+      "description": "Sales Tax",
+      "type": "Percentage",
+      "value": 5,
+      "prorate":"Not prorated",
+      "relationToTotal":"In addition to"
+    }
+  ],
+  "adjustmentsTotal": 50,
```

Comment:
adjustment total is incorrect, must be 55

Reply:
Done